pygmt/_show_versions.py

```diff
@@ -67,11 +69,12 @@ def _get_ghostscript_version() -> str | None:
     return None
 
 
-def _check_ghostscript_version(gs_version: str) -> str | None:
+def _check_ghostscript_version(gs_version: str | None) -> str | None:
     """
     Check if the Ghostscript version is compatible with GMT versions.
     """
-    from packaging.version import Version
+    if gs_version is None:
+        return None
```

Comment:
If ghostscript version cannot be detected, should there be a warning raised here?

Reply:
See https://github.com/GenericMappingTools/pygmt/pull/3376/files#r1706342731